test/controllers/VatObligationsControllerSpec.scala

```diff
@@ -100,11 +101,13 @@ class VatObligationsControllerSpec extends SpecBase with MockVatObligationsServi
   val testVrn: String = "555555555"
   val badVrn: String = "55"
 
+  val controllerComponents: ControllerComponents = stubControllerComponents()
```

Comment:
Could you put `controllerComponents` into `SpecBase` so it can be reused both here and `AuthActionImplSpec`.